client/.env.development

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_SERVER_URL=
+NEXT_PUBLIC_AUTH_URL=
```

Comment:
Can we get rid of `NEXT_PUBLIC_AUTH_URL`?  
I think it's confusing.

Reply:
I think you misunderstood `AUTH_URL`, look at the comments below.